org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/JDTLanguageServer.java

```diff
@@ -789,7 +791,22 @@ public void projectConfigurationUpdate(TextDocumentIdentifier param) {
 	public CompletableFuture<BuildWorkspaceStatus> buildWorkspace(boolean forceReBuild) {
 		logInfo(">> java/buildWorkspace (" + (forceReBuild ? "full)" : "incremental)"));
 		BuildWorkspaceHandler handler = new BuildWorkspaceHandler(pm);
-		return computeAsyncWithClientProgress((monitor) -> handler.buildWorkspace(forceReBuild, monitor));
+		return computeAsyncWithClientProgress((monitor) -> {
```

Comment:
Actually, I'd rather prefer we had a listener for when the default IVMInstall changes, that would take care of updating relevant gradle projects

Reply:
Fixed.